webapp/resources/templates/casConsentReviewView.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout" layout:decorate="~{layout}">
+
+<head>
+    <title th:text="#{screen.consent.review.header}"></title>
+
+    <script th:src="@{#{webjars.datatables.jquery.datatablesmin.js}}"></script>
+    <script th:src="@{#{webjars.datatables.datatables.bootstrapmin.js}}"></script>
+    <script type="text/javascript" th:inline="javascript">
+        var urls = {
+            delete: /*[[@{/consent/deleteConsentDecision}]]*/,
+            getConsentDecisions: /*[[@{/consent/getConsentDecisions}]]*/
+        };
+
+        function jqueryReady() {
+            head.load(
+                /*[[@{/js/consentReview.js}]]*/
+            );
+        }
+    </script>
+</head>
+
+<body id="cas">
+    <div layout:fragment="content">
+    <link rel="stylesheet" th:href="@{/css/consentReview.css}"/>
```

Comment:
This will likely need to go away at some point too and turned into sass. Can help with this bit if you want.

Reply:
It's [here](https://github.com/apereo/cas/pull/2934/files#diff-1c582a57f1e630723bf9bd25e2926c73), no?

Reply:
Ah, you're right. Never mind :)